### PR TITLE
Install Font Awesome 5

### PR DESCRIPTION
**Pull request by hokwaichan**
# Ticket Link

[1792](https://uhawaii.atlassian.net/browse/GROUPINGS-1792)

# List of squashed commits

-   Install Font Awesome 5

# Test Checklist

-   [x] Unit Tests Passed
-   [x] Integration Tests Passed
-   [x] General Visual Inspection

### CODE REVIEW

**Comment by JorWo** [comment]
These icons are also FontAwesome but we decided to download them as SVGs to import into to project.
image
Could you also replace id-card-solid.svg, key-solid.svg, user-solid.svg, and wrentch-solid.svg stored in /public? Thanks.

**Comment by JorWo** [comment]
Now that the GroupingsTable is merged, could you also update that too? Thanks a lot.

File: ui/src/app/(home)/_components/after-login.tsx [hunk 1]
```diff
@@ -73,12 +67,12 @@ const AfterLogin = async () => {
                                 className="flex justify-center items-center rounded-full
                             h-[100px] w-[100px] bg-white mx-auto relative lg:ml-0"
                             >
-                                <Image src="/uhgroupings/user-solid.svg" alt="user-solid" width={56} height={64} />
+                                <FontAwesomeIcon style={{ width: '56px', height: '64px' }}aria-label="user" icon={faUser} />
```
Comment (JorWo) [inline ui/src/app/(home)/_components/after-login.tsx:70]: Could you use className instead of style? Thanks.

**Comment by hokwaichan** [comment]
> Now that the GroupingsTable is merged, could you also update that too? Thanks a lot.

Would you also like to change the one in the dropdown menu and grouping-path-cell?

**Comment by JorWo** [comment]
> > Now that the GroupingsTable is merged, could you also update that too? Thanks a lot.
> 
> Would you also like to change the one in the dropdown menu and grouping-path-cell?

Yes please, thanks.

**Comment by hokwaichan** [comment]
> > > Now that the GroupingsTable is merged, could you also update that too? Thanks a lot.
> > 
> > 
> > Would you also like to change the one in the dropdown menu and grouping-path-cell?
> 
> Yes please, thanks.

For the dropdown menu I couldn't find the buttons on the page, is there a way to display them?

**Comment by JorWo** [comment]
Sorry, I meant this icon: 
image

**Review by JorWo — APPROVED** [review]
Thanks for updating the rest of the icons.

**Pull request merged**